.github/workflows/security-scan.yaml

```diff
@@ -0,0 +1,17 @@
+name: Security scan
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - hotfix/*
+      - work/secscan  # For development
+
+jobs:
+  python-scans:
+    name: Scan Python project
```

Comment:
out of curiosity what about the snap?

Reply:
These scanners sadly don't have the native ability to scan snaps, so until our self-hosted runners get access to security's API we're going to depend on occasional manual scans using the script at https://github.com/canonical/starscan